src/confidential-transactions/confidential-transactions.service.ts

```diff
@@ -100,26 +101,29 @@ export class ConfidentialTransactionsService {
   public async senderAffirmLeg(
     transactionId: BigNumber,
     body: SenderAffirmConfidentialTransactionDto
-  ): ServiceReturn<ConfidentialTransaction> {
+  ): Promise<{
+    result: Awaited<ServiceReturn<ConfidentialTransaction>>;
+    proofs: ConfidentialProofModel[];
```

Comment:
I added it in the response like this since grabbing proofs from the entity requires SQ to sync, so the response time could be slowed down waiting